port/unix_include/omrportpg.h

```diff
@@ -64,6 +64,10 @@ typedef struct OMRSTFLECache {
 } OMRSTFLECache;
 
 typedef struct OMRPortPlatformGlobals {
+#if defined(LINUX)
+	uintptr_t huge_pages_mmap_enabled;
+	uintptr_t memfd_call;
+#endif /* defined(LINUX) */
```

Comment:
Should these variables be at the beginning? There is a section defined for Linux already closer to the end of struct

Reply:
True, I'll move them there